src/components/tx/ApprovalEditor/index.tsx

```diff
@@ -1,30 +1,23 @@
 import { SafeTxContext } from '@/components/tx-flow/SafeTxProvider'
 import { createMultiSendCallOnlyTx, createTx } from '@/services/tx/tx-sender'
-import { Alert, Box, Divider, Skeleton, SvgIcon, Typography } from '@mui/material'
+import { Alert, Box, Skeleton, Typography } from '@mui/material'
 import { type SafeTransaction } from '@safe-global/safe-core-sdk-types'
 import { useContext } from 'react'
 import css from './styles.module.css'
 import { ApprovalEditorForm } from './ApprovalEditorForm'
 import { updateApprovalTxs } from './utils/approvals'
 import { useApprovalInfos } from './hooks/useApprovalInfos'
 import { decodeSafeTxToBaseTransactions } from '@/utils/transactions'
-import EditIcon from '@/public/images/common/edit.svg'
-import commonCss from '@/components/tx-flow/common/styles.module.css'
 import Approvals from '@/components/tx/ApprovalEditor/Approvals'
 import { type EIP712TypedData } from '@safe-global/safe-gateway-typescript-sdk'
 
 const Title = () => {
   return (
-    <div className={css.wrapper}>
-      <div className={css.icon}>
-        <SvgIcon component={EditIcon} inheritViewBox fontSize="small" />
-      </div>
-      <div>
-        <Typography fontWeight={700}>Approve access to</Typography>
-        <Typography variant="body2">
-          This allows contracts to spend the selected amounts of your asset balance.
-        </Typography>
-      </div>
+    <div>
+      <Typography fontWeight={700}>Allow access to tokens?</Typography>
+      <Typography variant="body2">
+        It will give access to third parties to use and spend the set amount of your tokens.
```

Comment:
"It will give" sounds a bit confusing imo as its not clear what "it" is. I would suggest keeping the previous text or slightly adjust it e.g. "This allows the spender to spend the specified amount of your tokens."

Reply:
It mean to be 'Access will give...', as an answer to the title.  Don't mind to rephrase the old one like you suggested!